modules/registry-refactoring/src/main/java/org/apache/airavata/registry/core/entities/appcatalog/ApplicationInterfaceEntity.java

```diff
@@ -60,6 +60,10 @@
 	@Column(name="HAS_OPTIONAL_FILE_INPUTS")
 	private boolean hasOptionalFileInputs;
 
+	@ElementCollection
+	@CollectionTable(name="APPLICATION_MODULE", joinColumns = @JoinColumn(name="MODULE_ID"))
+	private List<String> applicationModules;
```

Comment:
@machristie There is no corresponding column for this attribute in the APPLICATION_INTERFACE table. Also, it is of List<String> type. Can you please verify if the `@ElementCollection` annotation required here?

Reply:
I'm not sure, where did this even come from?  You could do something like this but you would need to use the APP_MODULE_MAPPING table.

Reply:
applicationModules is in the Thrift module but not in the APPLICATION_INTERFACE table. So when we update the ApplicationInterfaceEntity, this field won't get persisted in the database.

Reply:
This way was just a suggestion by Sachin.